mantidimaging/core/fitting/fitting_functions.py

```diff
@@ -8,6 +8,8 @@
 import numpy as np
 from scipy.special import erf
 
+FittingRegionType = tuple[float, float, float, float]
```

Comment:
Nice use of a type alias! If the elements in the tuple have meaning, would using a `NamedTuple` or a `dataclass` improve clarity, maybe something like this:
```Python
class FittingRegion(NamedTuple):
    x_min: float
    x_max: float
    y_min: float
    y_max: float
```